packages/pigeon/platform_tests/test_plugin/example/ios/RunnerTests/RunnerTests.swift

```diff
@@ -27,4 +30,30 @@ class RunnerTests: XCTestCase {
     let copy = MessageSearchReply.fromList(dict)
     XCTAssertEqual(reply.error, copy?.error)
   }
+
+  func callEchoStringFromProtocol() throws {
```

Comment:
To be a test, this has to start with `test`. Otherwise it's assumed by Xcode to be a helper method, and it won't be called.